### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by Denton-L**
Brought over some old changes left behind and made some minor corrections.

### CODE REVIEW

**Comment by axic** [comment]
I think it should be noted that now the build step is required for Chrome extensions.

**Comment by Denton-L** [comment]
Maybe mention a section on building it yourself and using the prebuilt version on `gh-pages`?

**Comment by axic** [comment]
There is the section "Building" (though it could use some more explanation). The Chrome extension part needs to explicitly say that it must be built before loading as an extension.

**Comment by Denton-L** [comment]
Alright, I added a section about building.

File: README.md [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
 
-    npm install    # fetch dependencies
+	npm install    # fetch dependencies
 	npm run build  # build application into build/app.js
 
-Now point your browser to `index.html` to open the application.  
+Now point your browser to `index.html` to open the application.
 
-# To use it as a chrome extension
+## Usage as as Chrome Extension
```
Comment (chriseth) [inline README.md]: Typo: `as as`

**Comment by Denton-L** [comment]
Typo corrected

**Pull request merged**